tests/js/spec/views/projectsDashboard/noProjectMessage.spec.jsx

```diff
@@ -55,4 +55,20 @@ describe('NoProjectMessage', function() {
       wrapper.find('Button[to="/organizations/org-slug/projects/new/"]')
     ).toHaveLength(1);
   });
+
+  it('handles loading projects from props', function() {
+    const lightWeightOrg = TestStubs.Organization();
+    delete lightWeightOrg.projects;
+
+    const child = <div>child</div>;
+
+    const wrapper = shallow(
+      <NoProjectMessage projects={[]} loadingProjects organization={lightWeightOrg}>
```

Comment:
Should we make sure lightweight version doesn't cause an unmount in a test?

Reply:
good idea, just added a test, let me know if that's the correct way to check for unmount/remount

Reply:
Hmmm, it seems correct, but I would double check it by stashing changes to `lightWeightNoProjectMessage.tsx` and seeing if it fails?

Reply:
good call changed to use a class component with jest functions inside the component's lifecycle methods.